dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -1517,6 +1517,12 @@ components:
             $ref: '#/components/schemas/WorkflowVersion'
           type: array
           uniqueItems: true
+        workflowVersionsOverride:
```

Comment:
If I understand correctly, this is only used internally so should not be exposed to API users